src/types/model.ts

```diff
@@ -95,27 +96,32 @@ export interface RegistryBlock {
   hash: string;
 }
 
+type OrdersPerOwner = Map<Owner, Set<ConditionalOrder>>;
+
+const logger = getLogger("Registry");
```

Comment:
Why registry if this is in `model.ts`?

Reply:
Its a similar comment as above. You took it out of the registry and has still a very generic name. 
I don't think getting loggers should be the memory leak (i hope)

Reply:
Got you.
Created a separate logger:
```
const loadOwnerOrdersLogger = getLogger("loadOwnerOrders");
```